code/Example/Drasil/GlassBR/DataDefs.hs

```diff
@@ -42,7 +42,7 @@ risk_eq = ((sy sflawParamK) /
 
 -- FIXME [4] !!!
 risk :: QDefinition
-risk = mkDataDef' risk_fun risk_eq (aGrtrThanB +:+ hRef +:+ ldfRef +:+ jRef) [sourceref $ S "[4]"]
+risk = mkDataDef' risk_fun risk_eq (aGrtrThanB +:+ hRef +:+ ldfRef +:+ jRef) [] [sourceref $ S "[4]"]
```

Comment:
This `sourceref $ S "[4]"` is the kind of thing we'll also want to change, at least eventually. I'll let it go on this PR, it can be caught in a later pass.